en/concept/custom_mixer_payload.md

```diff
@@ -0,0 +1,77 @@
+
+
+## Payload Mixer Example
+
+A payload mixer is just a [simple mixer](../concept/mixing.md#simple-mixer) that maps any of the function values from [Control Group #6 (First Payload)](../concept/mixing.md#control_group_6) to a particular output.
+You can then publish uORB topics to the selected control group function and their value will be mapped to the specified output.
+
+For this example, we'll create a custom mixer based on the *RC passthrough mixer* ([pass.aux.mix](https://github.com/PX4/Firmware/blob/master/ROMFS/px4fmu_common/mixers/pass.aux.mix)).
+This mixer is commonly loaded into the AUX PWM ports on large multicopters). 
+It passes through the values of 4 user-defined RC channels (set using the [RC_MAP_AUXx/RC_MAP_FLAPS](../advanced/parameter_reference.md#RC_MAP_AUX1) parameters) to the first four outputs on the AUX PWM output.
+
+```
+# Manual pass through mixer for servo outputs 1-4
+
+# AUX1 channel (select RC channel with RC_MAP_AUX1 param)
+M: 1
+S: 3 5  10000  10000      0 -10000  10000
+
+# AUX2 channel (select RC channel with RC_MAP_AUX2 param)
+M: 1
+S: 3 6  10000  10000      0 -10000  10000
+
+# AUX3 channel (select RC channel with RC_MAP_AUX3 param)
+M: 1
+S: 3 7  10000  10000      0 -10000  10000
+
+# FLAPS channel (select RC channel with RC_MAP_FLAPS param)
+M: 1
+S: 3 4  10000  10000      0 -10000  10000
+```
+
+> **Note** The file defines a [simple mixer](../concept/mixing.md#simple-mixer) with four outputs.
+> - `M: 1` indicates an output that is defined by one control input (the following `S` line).
+> - `S: 3 `_`n`_ indicates that the input is the n<>th<> input of [Control Group 3 (Manual Passthrough)](../concept/mixing.md#control-group-3-manual-passthrough).
+>  So for `S: 3 5` the input is called "RC aux1", and will output the value of the channel mapped by parameter `RC_MAP_AUX1`.
+> - The order of output declaration of 
+
+
+Start by copying the mixer file and putting it onto the SD Card at: **/etc/mixers/pass.aux.mix** (see [Mixing and Actuators > Loading a Custom Mixer](../concept/mixing.md#loading_custom_mixer).
+
+Remove the first section with a payload control group function input:
+- Change this:
+  ```
+  # AUX1 channel (control group 3, RC CH5) (select RC channel with RC_MAP_AUX1 param)
+  M: 1
+  S: 3 5  10000  10000      0 -10000  10000
+  ```
+- To:
+  ```
+  # Payload 1 (control group 6) channel 1
+  M: 1
+  S: 6 1  10000  10000      0 -10000  10000
+  ```
+
+Because this output is in the first position in the file it will map to the first AUX PWM output (unless UAVCAN is enabled).
+This output will now respect updates to the payload control group (6) output 1.
+
+Putting an output on group 6 works by publishing actuator control group 6. First you have to create the publication.
+This should happen once when the PX4 module is initialized (look for places where this pattern is already being used):
+```
+uORB::Publication<actuator_controls_s> _actuator_controls_pub{ORB_ID(actuator_controls_0)};
```

Comment:
@bkueng This confuses me. How does the system know I'm intersted in group 5? Is the last param a typo - ie should it be `actuator_controls_6`?

Reply:
You're right, it should be.
But 6 is not even implemented: https://github.com/PX4/Firmware/blob/master/msg/actuator_controls.msg#L17.

Reply:
@bkueng So I think you're saying that there is no way to publish to this topic - ie this guide is pointless? Correct?

What do I do about that then? Can you publish the topic, or do we go back to Lorenz or what?

Reply:
I think it's simple enough that we can just add the additional required steps:
- define the topic in https://github.com/PX4/Firmware/blob/master/msg/actuator_controls.msg#L17
- `uint8 NUM_ACTUATOR_CONTROL_GROUPS = 5`
- subscribe the topic in https://github.com/PX4/Firmware/blob/master/src/lib/mixer_module/mixer_module.cpp#L52

That would make it control group #4 though. And maybe some more bits are required.